database/db.dbm

```diff
@@ -4,7 +4,7 @@ CAUTION: Do not modify this file unless you know what you are doing.
  Unexpected results may occur if the code is changed deliberately.
 -->
 <dbmodel pgmodeler-ver="1.1.5" use-changelog="false" max-obj-count="18"
-	 last-position="147,221" last-zoom="1" scene-rect="0,0,1570.8,1076.8"
+	 last-position="257,118" last-zoom="1" scene-rect="0,0,1570.8,1076.8"
```

Comment:
What does this do?  I don't really understand what's going on in this file, but he rest of it looks pretty sensible.  This line's kind of mysterious though.

Reply:
This file is created by pgModeler app that I use to design DB schema.